.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Synchronize the kata-containers installation with in the host filesystem.
+#
+set -x
```

Comment:
How about `set -e` (and maybe `set -o nounset`) for all the scripts on this PR?

Reply:
Done. I had to add -O to rsync since it was failing to update the time attributes of some files.